Cargo.lock

```diff
@@ -4104,11 +4166,13 @@ dependencies = [
  "arena",
  "cfg-if",
  "log",
+ "md-5",
  "rustc_data_structures",
  "rustc_index",
  "rustc_macros",
  "scoped-tls",
  "serialize",
+ "sha-1 0.8.2",
```

Comment:
`pest_meta` already depends on a version of `sha-1` (0.7.0). Could you please depend on the same version here to avoid duplicating `sha-1`, `digest` and `generic-array`? Also `md-5` depends on the newer `digest`. Another option would be to update `pest_meta` to `2.1.3`, which depends on `sha-1` version `0.8` instead.

Reply:
I went with upgrading `pest_meta`, which seemed to clean up the lock file.